app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,26 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%= link_to "Sign up", new_registration_path(resource_name) %><br />
+<% end %>
+
+<%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+<!--  <%#= link_to "Forgot your password?", new_password_path(resource_name) %><br />-->
+  <%= link_to "Forgot your password?", "#" %><br />
```

Comment:
question: Why does this not point to the reset password path?

Reply:
I haven't included the reset password feature in this PR. That's why I disabled it